src/components/presentational/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,139 @@
+                    <div className="mdhui-line-tooltip-date">{format(payload[0].payload.date, "hh:mm:ss aaa")}</div>
+                </div>
+            );
+        }
+        return null;
+    };
+
+    const XAxisTick = ({ x, y, stroke, payload }: any) => {
+        var displayX = format(payload.value, "h aaa");
+        return <text fill="var(--mdhui-text-color-2)" x={x} y={y + 15} textAnchor="middle" fontSize="12">{displayX}</text>;
+    }
+
+    const YAxisTick = ({ x, y, stroke, payload }: any) => {
+        var display = Math.round(payload.value);
+        return <text fill="var(--mdhui-text-color-3)" x={x - 5} y={y + 5} textAnchor="end" fontSize="12">{display}</text>;
+    }
+
+    function getPercent(numerator: number): number {
+        const denominator = axis?.yRange ? axis.yRange[1] : 100;
+        return (numerator / denominator) * 100;
+    }
+
+    function createStopsFromThresholds() {
+        const y = axis?.yRange ? axis.yRange[1] : 100;
+        let stops: any = [];
+        var lineColor: string = defaultLineColor;
+        var thresholds = props.thresholds ?? [];
+
+        stops.push(<stop offset="0%" stopColor={lineColor} />);
+        for (var i = 0; i < thresholds.length; i++) {
+            if (y >= thresholds[i].value) {
+                lineColor = resolveColor(layoutContext.colorScheme, thresholds[i].overThresholdLineColor) || defaultLineColor;
+                var offSet = getPercent(thresholds[i].value);
+                stops.push(<stop offset={`${offSet}%`} stopColor={lineColor} />);
+            }
+        }
+        stops.push(<stop offset="100%" stopColor={lineColor} />);
+
+        return stops;
+    }
+
+    function standardChartComponents() {
+        return <>
+            <defs>
+                {props.thresholds &&
+                    <linearGradient id="thresholds" key={thresholdKey} x1="0%" y1="100%" x2="0%" y2="0%">
+                        {createStopsFromThresholds()}
+                    </linearGradient>
+                }
+
+                {!props.thresholds &&
+                    <linearGradient id="gradient" x1="0" y1="0" x2="0" y2="100%">
+                        <stop offset="0%" stopColor={defaultLineColor} />
+                        <stop offset="100%" stopColor={defaultLineColor} />
+                    </linearGradient>
+                }
+            </defs>
+            <Tooltip wrapperStyle={{ outline: "none" }} content={<GraphToolTip />} />
+            <CartesianGrid vertical strokeDasharray="2 4" />
+            {(props.thresholds)?.filter(t => t.referenceLineColor)?.map((threshold, index) =>
+                <ReferenceLine key={`threshref_${index}`} y={threshold.value} stroke={resolveColor(layoutContext.colorScheme, threshold.referenceLineColor)} />
+            )}
+            <YAxis tick={YAxisTick} type='number' domain={axis?.yRange ?? [0,100]} />
+            <XAxis tick={XAxisTick} axisLine={true} dataKey="date" tickMargin={0} minTickGap={0} tickCount={24}
```

Comment:
This is where I think you need to specify the "interval" to get the axis ticks to show up at 3/6/9 etc.